functions/zoom-downloader.py

```diff
@@ -131,33 +132,33 @@ def host_name(self):
                                 )
         return self._host_name
 
+    @property
+    def uuid(self):
+        return self.data["uuid"]
+
+    @property
+    def zoom_series_id(self):
+        return self.data["zoom_series_id"]
+
     @property
     def recording_files(self):
         if not hasattr(self, "_recording_files"):
+
             files = self.data["recording_files"]
-            start_times = sorted(
-                set([file["recording_start"] for file in files])
-            )
-            track_numbers = {
-                start_times[i]: i for i in range(len(start_times))
-            }
+            start_times = sorted(set(
+                [ file["recording_start"] for file in files ]
```

Comment:
The spaces around the brackets are a linter error for me

Reply:
how have I done this for so long without the pylint plugin installed?